charts/timescaledb-single/templates/configmap-scripts.yaml

```
@@ -332,7 +332,20 @@ data:
     __SQL__
     done
 
-    # We always exit 0 this script, otherwise the database initialization fails.
+    # This directory may contain user defined post init steps
+    for file in $(find {{ template "post_init_dir" . | quote }} -! type d ! -name '.*' | sort)
+    do
+        log "Calling post init script $file"
+        "{{ template "post_init_dir" . }}/$file" "$@"
```

Here you could add support for multiple file types with something like:
```suggestion
      [[ ! -s "$file" ]] && continue

      case "$file" in
        *.sh)
          if [ -x "$file" ]; then
            log "Call post init script [ $file ]"
            "$file" $@
            EXITCODE=$?
          else
            log "Source post init script [ $file ]"
            . "$file"
            EXITCODE=$?
          fi
          ;;
        *.sql)
          log "Apply post init sql [ $file ]"
          # Disable synchronous_commit since we're initializing
          PGOPTIONS="-c synchronous_commit=local" psql -d "$URL" -f "$file"
          EXITCODE=$?
          ;;
        *.sql.gz)
          log "Decompress and apply post init sql [ $file ]"
          gunzip -c "$file" | PGOPTIONS="-c synchronous_commit=local" psql -d "$URL"
          EXITCODE=$?
          ;;
        *)
          log "Ignore unknown post init file type [ $file ]"
          EXITCODE=0
          ;;
      esac
```

---

Yeah, that would improve things a bit, I especially like the `*.sh` and `*.sql` distinction.